src/tlo/methods/tb.py

```diff
@@ -2379,6 +2398,10 @@ def apply(self, person_id, squeeze_factor):
                 ACTUAL_APPT_FOOTPRINT = self.make_appt_footprint(
                     {"TBFollowUp": 1, "LabTBMicro": 1, "LabMolec": 1}
                 )
+
+                self.EQUIPMENT = {'Sputum Collection box', 'Ordinary Microscope'}
+                self.EQUIPMENT = {'Sputum Collection box', 'Gene Expert (16 Module)'}
+
```

Comment:
(1) This overwrites the equipment used for the sputum test above. If all equipment should be used - for the sputum test & the xpert test, there needs to be `self.EQUIPMENT.update({'Sputum Collection box', ...} )` instead. Is this so?

(2) Here are two declarations for xpert test, which one is correct?

Reply:
In this case, both a sputum microscopy and genexpert was used, so both are declared separately. There are also two diagnostic appts declared ("LabTBMicro": 1, "LabMolec": 1)

Reply:
This is pushed now, in a single line for clarity

Reply:
Is what you meant by this: 

If sputum test is performed -> equipment used {'Sputum Collection box', 'Ordinary Microscope'}

And if that sputum test returned positive and person is not diagnosed with mdr, xpert test is performed -> equipment used {'Sputum Collection box', 'Gene Expert (16 Module)'}
(so you were trying here to join equipment used for these two tests?)

If so, then what I just suggested would do that work - the update keeps the equipment already added for the sputum test and updates the set of used equipment by adding the equipment for xpert test.

Reply:
I think that does exactly the same thing..?

If there's a style preference then please go ahead and make the change.

Reply:
the edit should be right now

Reply:
Brilliant, thank you!